puresnmp/test/__init__.py

```diff
@@ -74,4 +95,4 @@ def readbytes(filename):
             line = line.split(':')[1]
         str_bytes.extend(line.split())
     values = [int(char, 16) for char in str_bytes]
-    return bytes(values)
+    return bytearray(values)
```

Comment:
Why did you replace `bytes` with `bytearray` here? `bytes` also exists in Python2.

Reply:
... or to be more precise, there is a tiny difference in `bytes` and `bytearray`. Was there a reason you needed a `bytearray` instead of `bytes`? I don't think it does make a big difference here so I am fine either way. I'm just curious.